src/tests/test_integration.py

```diff
@@ -954,3 +964,15 @@ def foo():
     _, _, code = env.invoke()
 
     assert code == 0
+
+
+def test_syntax_error_multiple_files(env):
```

Comment:
Perhaps add a third _valid_ file and see that it's parsed correctly? (or is that TODO in the issue? Why not do it now?)